### PR TITLE
Add instantiation to "structs" example

### PR DESCRIPTION
**Pull request by MegaRedHand**

### CODE REVIEW

File: content/en/examples/structs.md [hunk 1]
```diff
@@ -26,3 +26,63 @@ struct Company {
...
 }
 ```
+
+You can instantiate structs via the following syntax:
+
+```rust {.codebox}
+let employee = Employee {
+    age: 53,
+    id: 1616,
+    role: 'asdas',
```
Comment (matias-gonz) [inline content/en/examples/structs.md:36]:
```suggestion
    role: 'engineer',
```

File: content/en/examples/structs.md [hunk 2]
```diff
@@ -26,3 +26,63 @@ struct Company {
...
+};
+```
+
+If you have variables named the same as the members, you can use a shorter syntax:
+
+```rust {.codebox}
+let age = 53;
+let id = 1616;
+let role = 'asdas';
```
Comment (matias-gonz) [inline content/en/examples/structs.md:45]:
```suggestion
let role = 'engineer';
```

File: content/en/examples/structs.md [hunk 3]
```diff
@@ -26,3 +26,63 @@ struct Company {
...
+struct Company {
+    employees: Array<Employee>,
+    number_of_buildings: u32
+}
+
+fn main() {
+    let employee = Employee {
+        age: 53,
+        id: 1616,
+        role: 'asdas',
```
Comment (matias-gonz) [inline content/en/examples/structs.md:76]:
```suggestion
        role: 'engineer',
```

**Review by rcatalan98 — COMMENTED** [review]
A nice addition would be to apply the same changes to the Spanish files. This way we can remain consistent.

**Pull request merged**